.github/workflows/ci.yml

```diff
@@ -31,18 +31,15 @@ jobs:
         with:
           python-version: '3.x'
 
-      - name: Install python-build and twine
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install build twine
-          python -m pip list
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v3
 
       - name: Build v0.0.1 wheel and sdist
         run: |
-          python -m build --outdir ./dist _action_path/tests/test_package
+          uv tool run --from build pyproject-build --installer=uv --outdir ./dist _action_path/tests/test_package
```

Comment:
I do know that `uvx` exists and is an alias for `uv tool run`, but I decided to keep it more explicit for now.

Reply:
yep also not sure about their plans in the future so fine with that 👍